legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -182,16 +183,23 @@ def search_businesses():
         for draft_dao in draft_query.all():
             draft = {
                 'identifier': draft_dao.temp_reg,  # Temporary registration number of the draft entity
-                'legalType': draft_dao.json_legal_type  # Legal type of the draft entity
+                'legalType': draft_dao.json_legal_type,  # Legal type of the draft entity
+                'draftType': Filing.FILINGS.get(draft_dao.filing_type, {}).get('temporaryCorpTypeCode'),
```

Comment:
OK, so this will return TMP/ATMP/RTMP/CTMP, right?

Reply:
yes